webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -56,32 +57,33 @@ const TaskActionDropdown: React.FC<Props> = ({ task }: Props) => {
             experimentId: parseInt(task.id),
```

Comment:
chore: `setExperimentState` is defined as:
```
export const setExperimentState =
  async ({ state, ...rest }: PatchExperimentState): Promise<void> => {
    return patchExperiment({
      body: { state },
      ...rest,
    });
  };
```
It's missing `await` in front of `patchExperiment`.

Reply:
leaving it to https://github.com/determined-ai/determined/pull/739